src/invidious/routes/API/v1/channels.cr

```diff
@@ -0,0 +1,267 @@
+class Invidious::Routes::V1Api < Invidious::Routes::BaseRoute
+  def home(env)
+    locale = LOCALES[env.get("preferences").as(Preferences).locale]?
```

Comment:
The locale shouldn't be taken into account in the API (it has to be stable for programmatic use)

Reply:
That was present before the extraction, should I remove it? I'd rather not do any noticable changes to the API with this PR — especially when it is a "breaking" change like this.

Reply:
Double check to be sure, but that shouldn't break anything. Moreover, it's already done in the `before_all` handler: https://github.com/iv-org/invidious/blob/master/src/invidious.cr#L287

Reply:
The problem comes when you take into account just how intertwined the backend and frontend of Invidious is.

Reply:
The problem comes when you take into account just how intertwined the backend and frontend of Invidious is. Take the `template_*` and `fetch_*` methods for comments:
 <details>

https://github.com/iv-org/invidious/blob/bc937a6434eaeb231c7621426fcf818fe07807fb/src/invidious/comments.cr#L186
https://github.com/iv-org/invidious/blob/bc937a6434eaeb231c7621426fcf818fe07807fb/src/invidious/comments.cr#L301-L302
https://github.com/iv-org/invidious/blob/bc937a6434eaeb231c7621426fcf818fe07807fb/src/invidious/comments.cr#L370-L384

</details>

They all require the locale to translate **key features** of the comments (timestamps, permalink, reply loading, etc). 

Examples like those are littered everywhere throughout the API logic; and without significant changes — that is also out of scope for this PR — none of them can be removed.

Reply:
Ooooffff .... Ok, nevermind, lol

Reply:
Yep... In order to do any meaningful refactor of the API, we'll first need to do the backend split.